kernel/arch/dreamcast/kernel/cache.s

```diff
@@ -24,11 +24,15 @@
 ! taken from sh-stub.c.
```

Comment:
Not sure if this is the right comment now.
This has been completely rewritten in the last year.

Reply:
Updated